packages/create-svelte/bin.js

```diff
@@ -180,7 +176,15 @@ async function main() {
 	console.log(`  ${i++}: ${bold(cyan('npm run dev -- --open'))}`);
 
 	console.log(`\nTo close the dev server, hit ${bold(cyan('Ctrl-C'))}`);
-	console.log(`\nStuck? Visit us at ${cyan('https://svelte.dev/chat')}\n`);
+	console.log(
+		`\nNew? Do the tutorial at ${cyan('https://learn.svelte.dev/tutorial/introducing-sveltekit')}\n`
```

Comment:
I suggested above in the docs section "try" instead of "do". Personally I would just remove this line, but I'm noting the wording here just in case we keep it